ui/js/actions/content.js

```diff
@@ -505,3 +505,42 @@ export function doPublish(params) {
     });
   };
 }
+
+export function doAbandonClaim(claimId, name, txid, nout) {
+  return function(dispatch, getState) {
+    const state = getState();
+
+    dispatch({
+      type: types.ABANDON_CLAIM_STARTED,
+      data: {
+        claimId: claimId,
+      },
+    });
+
+    const errorCallback = error => {
+      dispatch(doOpenModal(modals.TRANSACTION_FAILED));
+    };
+
+    const successCallback = results => {
+      if (results.txid) {
+        dispatch({
+          type: types.ABANDON_CLAIM_SUCCEEDED,
+          data: {
+            claimId: claimId,
+          },
+        });
+        dispatch(doResolveUri(lbryuri.build({ name, claimId })));
+        dispatch(doFetchClaimListMine());
+      } else {
+        dispatch(doOpenModal(modals.TRANSACTION_FAILED));
```

Comment:
Probably better as something like `errorCallback(new Error("Claim abandon received unknown response signature"))`. If we made it to this code, it's quite possible the transaction succeeded, right?

Reply:
If the txn succeeds then `txid` _will be_ returned in the `results`. So if we make into the `else` code then, for some reason the transaction must have failed.

Reply:
I think this is mildly incorrect, but I'm going to merge it anyway. If say, the API signatures of the daemon change such that `txid` is no longer confirmed (suppose it is renamed to `transaction_id`), this would still be a successful transaction, but now report as a failed transaction.

In general, we ought to trust that if the daemon says it is a success in terms of the JSONRPC response, then it is a success.

If the success doesn't match the signature we expect, it's fine and good to panic/error, but the most likely explanation of this would be a change in protocol API signatures without a corresponding application update.